compose/release-notes.md

```diff
@@ -2113,8 +2113,8 @@ naming scheme accordingly before upgrading.
     You don't have to use it - your existing Compose files will run on Compose
     1.6 exactly as they do today.
 
-    Check the upgrade guide for full details:
-    https://docs.docker.com/compose/compose-file#upgrading
```

Comment:
This URL was not valid. We could not catch it with `htmlproofer` because not solved as link.

Reply:
That's a line in the file from 2016, probably there were changes in `compose-file` affecting this. Did we have htmlproofer then?
You say "DOCS_URL env var not being taken into account properly.", how should we take this var into account? 
I'm unaware of this topic. cc: @usha-mandya

Reply:
> You say "DOCS_URL env var not being taken into account properly.", how should we take this var into account?
> I'm unaware of this topic. cc: @usha-mandya

`DOCS_URL` has been added in https://github.com/docker/docker.github.io/pull/15250 to avoid hardcoded URLs that would not match the environment (stage vs lab vs prod). See for example the source of https://docs-stage.docker.com/ and you will see some URLs with `docs.docker.com` which is not right and should be `docs-stage.docker.com`.

tldr; this is an internal enhancement that should not impact your workflow.

Reply:
Got it, so to get right base URL according with environment. Thanks for the explanation.